### ISSUE TITLE
Speeding up html export of a many frames HoloMap

### ISSUE THREAD

**Issue by ahuang11** [pyviz/holoviews#3523]
Sometimes, a server is unavailable for DynamicMap, or other times, the DynamicMap's animation is a bit choppy due to buffering. So I was wondering if it's possible to parallelize this if this happened to loop over each frame? I looked at the source code, but I couldn't figure where the values are formatted into the html output.
https://github.com/pyviz/holoviews/blob/master/holoviews/plotting/renderer.py#L278 
https://github.com/pyviz/holoviews/blob/master/holoviews/plotting/bokeh/renderer.py#L106

```
import numpy as np
import holoviews as hv
hv.extension('bokeh')

frequencies = np.arange(0, 45, 0.5)

def sine_curve(phase, freq):
    xvals = [0.1* i for i in range(100)]
    return hv.Curve((xvals, [np.sin(phase+freq*x) for x in xvals]))

curve_dict = {f:sine_curve(0,f) for f in frequencies}

hmap = hv.HoloMap(curve_dict, kdims='frequency')
hv.save(hmap, 'test.html')
```

**Comment by philippjfr** [comment]
There will likely never be a solution that works in parallel here, particularly for bokeh. It has to evaluate the state space which means modifying the model and then recording the changes applied to it. It isn't possible to evaluate multiple state changes at the same time. Therefore I'm regretfully have to close this as wontfix.

**Comment by ahuang11** [comment]
So is it possible with matplotlib?

**Comment by philippjfr** [comment]
No, that was my mistake and I edited the answer. Exact same issue in matplotlib.
